README.md

```diff
@@ -337,7 +350,7 @@ $ node-debug --web-host 127.0.0.2 app
 ``` 
 Pass `--option=value` to debugging process:
 ```
-$ node-debug app --option value
+$ node-debug --option value app
```

Comment:
AFAIU, this example is illustrating how to pass arguments to the process being debugged. The proposed version shows how to pass arguments to Node Inspector instead, which is already covered by the previous examples with `-p` and `--web-host`.

Having said that, this is IMO a good place where to include the note "Make sure you put the Node Inspector options before the main script argument."